magefile.go

```diff
@@ -0,0 +1,38 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//go:build mage
+// +build mage
+
+package main
+
+import (
+	"github.com/magefile/mage/sh"
+
+	// mage:import
+	_ "github.com/elastic/elastic-agent/dev-tools/mage/target/common"
+	// mage:import
+	_ "github.com/elastic/elastic-agent-libs/dev-tools/mage"
+)
+
```

Comment:
Would be nice to copy these aliases here as well, so contributors use it consistently across all our repositories. It's already in Beats ([documented](https://www.elastic.co/guide/en/beats/devguide/current/beats-contributing.html#running-linter)) and elastic-agent-libs https://github.com/elastic/elastic-agent-libs/blob/df72f6afbb56a8f8547523f63fdd5cb0a8671d69/magefile.go#L29-L34